contracts/USM.sol

```diff
@@ -60,7 +61,6 @@ contract USM is IUSM, ERC20Permit, Delegable {
         returns (uint)
     {
         // First calculate:
-        require(fum.totalSupply() > 0, "Fund before minting");
```

Comment:
Hmmm I guess this sorta follows my own principle!  (Minimizing `require()`s.)  In this case I think this one adds clarity for anyone who tries to call `mint()` before `fund()`.  But I could see it either way - maybe your comment is enough.

Reply:
It's a call to an external contract, so it is quite expensive (1500 gas I think), for something that might happen only once and that would be caught anyway.

I like to be very clear in contracts, but in this case I prefer to save everyone gas, let's not do [this](https://twitter.com/Agusx1211/status/1313759898258362378).

Reply:
Ahh, very interesting.  OK fair enough!  Another reason to try to get math right rather than sprinkle `require()`s to handle incorrect math (y'know, when practical anyway).

Reply:
I'll keep the lesson here in mind for other cases too, I hadn't been thinking about this aspect.